lib/connection.go

```diff
@@ -32,6 +32,10 @@ const (
 	DefaultDecayDuration = 0
 )
 
+var (
```

Comment:
Previously we were using the global `http.DefaultClient` which had no timeout. Now we're using this one which we can set the timeout on just for ES.